src/ontology/mondo-ingest.Makefile

```diff
@@ -160,15 +160,20 @@ mappings: sssom $(ALL_MAPPINGS)
 #################
 # Utils #########
 #################
-# Documentation for this commands in this section is in: `docs/developer/ordo.md`
-
+# Documentation for `report-mapping-annotations` and `update-jinja-sparql-queries`: `docs/developer/ordo.md`
 report-mapping-annotations:
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/report_mapping_annotations.py
 
 update-jinja-sparql-queries:
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_replace_annotation_based_mappings.py
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_mapping_annotations_violation.py
 
+config/%_term_exclusions.txt: config/%_exclusions.tsv component-download-%.owl
+	python3 ../scripts/exclusion_term_expansion/exclusion_term_expansion.py \
+	--exclusions-path $(word 1,$^) \
+	--onto-path $(TMPDIR)/$@.owl \
```

Comment:
@matentzn 
## `.owl.owl` issue
Also, I think we spoke briefly about this before, but what's with the `.owl.owl` (e.g. `component-download-icd10cm.owl.owl`). Is this just a bug that needs to be fixed? That is, do we need to update several of our make commands such that we only produce/reference files with `.owl` rather than `.owl.owl`?

Reply:
As further indication that there is something strange with the `component-download-%.owl` recipe, notice that this is the only case where below I have to also list the directory (`$(TMPDIR)/`) and the additional file extension `.owl`. For the other dependencies, the name/path of the make recipe is the same name/path as the file generated.

```make
config/%_term_exclusions.txt: config/%_exclusions.tsv component-download-%.owl $(TMPDIR)/%_relevant_signature.txt
	python3 ../scripts/exclusion_term_expansion/exclusion_term_expansion.py \
	--exclusions-path $(word 1,$^) \
	--onto-path $(TMPDIR)/$(word 2,$^).owl \  # <---- the odd one
	--relevant-signature-path $(word 3,$^)
```